components/Table/DataRowSmall.tsx

```diff
@@ -15,12 +15,10 @@ const DataRowSmall: FC<DataRowProps> = ({ items }) => {
       bg={$bg}
       wrap="wrap"
       p="1rem"
+      onClick={onClick}
+      cursor={onClick ? 'pointer' : 'default'}
```

Comment:
Seeing a: `validateDOMNesting(...): <div> cannot appear as a child of <tbody>` warning on local and I think this element might be the culprit. I believe you can get around it by adding `as="tr"` but I'm not certain (see [this](https://chakra-ui.com/docs/components/recipes/as-prop) for more)